src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.closure.txt

```diff
@@ -60,7 +60,7 @@
    60|      1|            ||
    61|      1|            {
    62|      1|                let mut countdown = 0;
-   63|      1|                if is_false {
```

Comment:
Same problem as above, and below. I'm fairly sure that all of these coverage result changes will go away when you are using the latest version of LLVM 12 llvm-cov, with the reverted patch.